content/_partners/faq._en.md

```diff
@@ -130,7 +130,7 @@ development_accordion:
         id: c-5
         title: Do we need an approved IAA before we can launch our integration with Login.gov?
         content: >-
-            In order to launch your integration with Login.gov, your agency must first complete an IAA. You can test your application during the IAA process. Once testing is complete and the IAA has been executed, Login.gov aims to launch your integration within two weeks.  [Learn more about our IAA process](/partners/get-started/#iaa-anchor){:class="usa-nav_link caret"}
+            In order to launch your integration with Login.gov, your agency must first complete an IAA. You can test your application during the IAA process. Once testing is complete and the IAA has been executed, Login.gov aims to launch your integration within two weeks. We recommend a grace period between deployment and implementation on your site.  [Learn more about our IAA process](/partners/get-started/#iaa-anchor){:class="usa-nav_link caret"}
```

Comment:
One space after a period. We have two here.

Reply:
another great catch thank you! No typewriters here!